package.json

```diff
@@ -52,6 +52,7 @@
     }
   },
   "dependencies": {
+    "base64-js": "^1.5.1",
```

Comment:
Is this a pretty lightweight package?  It seems like converting a base64 encoded string to a bytearray would be pretty straightforward with normal javascript, but I could be wrong.  If that is a heavy package, we may want to just pull the relevant code into this repo.

Reply:
Quick search turned up this:

    Uint8Array.from(atob(base64_string), c => c.charCodeAt(0))

Link: https://stackoverflow.com/a/41106346/194065

Do we really need the separate package?  Want to keep to dependencies to a minimum whenever we can.

Reply:
`atob` and `btoa` functions are only available in the context of a web browser, but absent when running on device, so we would need to [polyfill](https://stackoverflow.com/a/51525605/4257223) in order to get access to those functions from React Native.  I was using the `base-64` package but @garyp [warned against](https://github.com/streem/react-native-twilio-video-webrtc/pull/16#discussion_r1369901058) using that due to some added complexities.

Gary also mentioned that we could use the `rfc4648` library as well.

Here is the size comparison of the two libraries on bundlephobia:
base64-js: 1.5 kb minified
rfc4648: 2.1kb minified

Reply:
Yeah, they're tiny. I don't think it's worth worrying about trying to vendor them into our code.

Also, `atob` and `btoa` should not be used, despite what SO says 😉 Those two functions were added to browsers long before Javascript supported binary types like `UInt8Array`. As a result, they use strings to emulate binary data (hence the need to use `c.charCodeAt(0)` to convert the output of `atob` to a real binary value). This mostly works but has pitfalls and is easy to screw up.

Reply:
I think we could use what our protobuf solution already provides for us here?

<img width="1728" alt="Screenshot 2023-10-26 at 2 40 00 PM" src="https://github.com/streem/react-native-twilio-video-webrtc/assets/30332608/dd091fa1-505d-4821-879f-ebaa96ca1a4e">

Reply:
Looking at the source code for that util, its just pure JS as @seanadkinson mentioned, so there shouldn't be a need to add a separate package here. We can use what we already have!
<img width="1728" alt="Screenshot 2023-10-26 at 2 42 59 PM" src="https://github.com/streem/react-native-twilio-video-webrtc/assets/30332608/b1a7a278-4e61-4235-88fa-f938c1366dd7">

Reply:
It doesn't make sense though for this Twilio Video library to have a dependency on a protobuf library.

Reply:
Oh you are 100% correct. I assumed this was still in the sdk.